client/lib/purchases/assembler.ts

```diff
@@ -63,6 +63,18 @@ function createPurchaseObject( purchase: RawPurchase | RawPurchaseCreditCard ):
 		isRenewal: Boolean( purchase.is_renewal ),
 		meta: purchase.meta,
 		priceText: purchase.price_text,
+		priceTierList: purchase.price_tier_list?.map(
+			( ptRaw ): PriceTier => ( {
+				minimumUnits: ptRaw.minimum_units,
+				maximumUnits: ptRaw.maximum_units,
+				minimumPrice: ptRaw.minimum_price,
+				maximumPrice: ptRaw.maximum_price,
+				minimumPriceDisplay: ptRaw.minimum_price_display,
+				maximumPriceDisplay: ptRaw.maximum_price_display,
+				minimumPriceMonthlyDisplay: ptRaw.minimum_price_monthly_display,
+				maximumPriceMonthlyDisplay: ptRaw.maximum_price_monthly_display,
+			} )
+		),
```

Comment:
Suggestion to make this more readable:
```
priceTierList: purchase.price_tier_list?.map( ( tier ) => mapRecordKeysRecursively( tier, snakeToCamelCase ) );
```

Reply:
TIL 👍

Reply:
On second thought I wonder if this is a batter approach since all of the type safety is lost. and the type is converted to a more generic Record<string,unknown> type. Going to stick to the existing approach. Let me know if you can think of an approach that can also preserve the types.